tests/integrations/raftstore/test_single.rs

```diff
@@ -92,7 +92,7 @@ fn test_delete() {
 }
 
 #[test_case(test_raftstore::new_node_cluster)]
-#[test_case(test_raftstore_v2::new_node_cluster)]
+// v2 doesn't support.
```

Comment:
Is it rocksdb delete range or RaftCmd::DeleteRange? Could you add some comments?